wandb/env.py

```diff
@@ -371,6 +374,54 @@ def get_use_v1_artifacts(env: Optional[Env] = None) -> bool:
     return val
 
 
+def get_async_upload_concurrency_limit(
```

Comment:
technically you could move most of this logic into settings

Reply:
Oooooh yes, this is better.